Docs/restructure and improve READMEs (#87)
ubilabs/google-maps-react-hooks: README.md

# Google Maps React Hooks
[![npm version](https://img.shields.io/npm/v/@ubilabs/google-maps-react-hooks)](https://www.npmjs.com/package/@ubilabs/google-maps-react-hooks) [![GitHub license](https://img.shields.io/badge/license-MIT-green.svg)](./LICENSE)

## Description

> This is a JavaScript library to easily implement a Google Maps map into your React application. It comes with a collection of React hooks to access the Google Maps map instance all over your components and to handle some common interactions with the map.

Reviewer: Maybe we can add/change the last sentence to sth. like:

```suggestion
This is a JavaScript library to easily implement a Google Maps map into your React application. It comes with a collection of React hooks to access the Google Maps map instance all over different components. It also provides different hooks to use [Google Map services](https://developers.google.com/maps/documentation/javascript/services) and handle common map interactions.
```

#### Table of contents

> - [Google Maps React Hooks Library](./library)

Reviewer: This not linking to the `README.md` but to the director itself.

```suggestion
- [Google Maps React Hooks Library](./library/README.md)
```

Same for all the links below...

Author: This was intended. If you open a directory that includes a README file, GitHub displays the content of the README right below the content of the directory.

Reviewer: Okay, thought it should just navigate between the READMEs. But then all good

> - [Examples](./examples)

Reviewer: ```suggestion
- [Examples](./examples/README.md))
```

Author: See comment above

>   - [Basic Google Map](./examples/basic-google-map)

Reviewer: ```suggestion
  - [Basic Google Map](./examples/basic-google-map/README.md)
```

and so on for the other one belows :)

Author: See comment above
